packages/datadog-instrumentations/src/confluentinc-kafka-javascript.js

```diff
@@ -0,0 +1,380 @@
+'use strict'
+
+const {
+  addHook,
+  channel,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+// Create channels for Confluent Kafka JavaScript
+const channels = {
+  producerStart: channel('apm:@confluentinc/kafka-javascript:produce:start'),
```

Comment:
@bengl fyi, it shouldn't be any sort of issue, but I hadn't considered that we'd need @ scoped packages in channel names.